src/marketing/price-rules-catalog-create.md

```diff
@@ -179,17 +181,25 @@ To apply the rule, do one of the following:
 - Click <span class="btn">Save and Apply</span>.
 - Click <span class="btn">Save</span>. Then from the Catalog Price Rules page, click <span class="btn">Apply Rules</span>.
 
+![Catalog price rules - pricing structures]({% link images/images/price-rule-catalog-saved.png %}){: .zoom}
+_Pricing Structure Rules_
+
 Price rules are automatically processed with other system rules each night. When you create a new price rule, allow enough time for it to get into the system. Then, test the rule to make sure that it works correctly. As new rules are added, Magento recalculates the prices and the priorities accordingly.
 
 <!--{% endif %}-->
 <!--{% if "Default.EE-B2B" contains site.edition %}-->
 ## Step 6: Save and Test the Rule
 
-1. When complete, click <span class="btn">Save Rule</span>.
+1. When complete, click <span class="btn">Save</span>.
+
+    The Rule Information page displays an updated timeline in the Scheduled Changes for the rule.
+
+    ![Catalog price rules - scheduled changes]({% link images/images-ee/price-rule-scheduled-changes-updated.png %}){: .zoom}
+    _Scheduled Changes_
 
-2. Test the rule to make sure that it works correctly.
+1. Test the rule to make sure that it works correctly.
 
-  Price rules are automatically processed with other system rules each night. When you create a new price rule, allow enough time for it to get into the system. Then, test the rule to make sure that it works correctly. As new rules are added, Magento recalculates the prices and the priorities accordingly.
+    Price rules are automatically processed with other system rules each night. When you create a new price rule, allow enough time for it to get into the system. Then, test the rule to make sure that it works correctly. As new rules are added, Magento recalculates the prices and the priorities accordingly.
```

Comment:
I'm not so sure about this statement. The rules on my devbox take effect immediately, though I often reindex frequently when working on staging. I don't know who to ask to verify this.

Reply:
I am not sure either. This information was already here, but might need to be updated.